ami_backup.py

```diff
@@ -2,98 +2,49 @@
 #
 # @author Robert Kozora <bobby@kozora.me>
 #
-# This script will search for all instances having a tag with "Backup" or "backup"
-# on it. As soon as we have the instances list, we loop through each instance
-# and create an AMI of it. Also, it will look for a "Retention" tag key which
-# will be used as a retention policy number in days. If there is no tag with
-# that name, it will use a 7 days default value for each AMI.
+# "retention_days" is environment variable which will be used as a retention policy number in days. If there is no
+# environment variable with that name, it will use a 14 days default value for each AMI.
 #
-# After creating the AMI it creates a "DeleteOn" tag on the AMI indicating when
+# After creating the AMI it creates a "ami_delete_on" tag on the AMI indicating when
```

Comment:
@comeanother all of our tags are camel case. Not sure why we are changing to snake case.